Library/Homebrew/cmd/info.rb

```diff
@@ -161,4 +169,11 @@ def decorate_dependencies(dependencies)
     end
     deps_status * ", "
   end
+
+  def decorate_requirements(requirements)
+    req_status = requirements.collect do |req|
+      req.satisfied? ? pretty_installed(req.display_s) : pretty_uninstalled(req.display_s)
```

Comment:
Anything wrong with keeping the `?:` if just pulling `req.display_s` out to a variable gets it under 80 characters?